app/models/miq_policy.rb

```diff
@@ -369,6 +370,38 @@ def self.seed
     end
   end
 
+  def self.seed_fixtures
+    fixture_file = File.join(FIXTURE_DIR, "miq_policies.yml")
+    fixtures = File.exist?(fixture_file) ? YAML.load_file(fixture_file) : []
+
+    fixtures.each do |policy_fixture|
+      rec = seed_policies(policy_fixture.except(:conditions))
```

Comment:
Should this be singular since we're only seeing one policy at a time?  Sorry.  😢